gatsby-node.esm.js

```diff
@@ -300,6 +305,13 @@ exports.createPages = async ({ actions, graphql }) => {
             data: pivotedData,
           },
         })
+        /*createPage({
+          path: `${path}/comingsoon/`,
+          component: ComingSoon,
+          context: {
+            data: pivotedData,
+          },
+        })*/ // Currently commented out as we don't want to generate these pages yet. TODO: @jtan re-enable.
```

Comment:
This should be at the same URL as the launched page. Missing this AC: 'For new verticals that have not yet launched, a there is a “Coming soon” page at the URL at the same URL as the future page.'